internal/image/live.go

```diff
@@ -93,6 +93,13 @@ func (img *LiveImage) InstantiateManifest(m *manifest.Manifest,
 		}
 		artifactPipeline = vmdkPipeline
 		artifact = vmdkPipeline.Export()
+	case platform.FORMAT_OVA:
+		vmdkPipeline := manifest.NewVMDK(m, buildPipeline, imagePipeline)
+		ovfPipeline := manifest.NewOVF(m, buildPipeline, vmdkPipeline)
+		archivePipeline := manifest.NewTar(m, buildPipeline, ovfPipeline, "archive")
```

Comment:
Shouldn't this be `artifactPipeline` and/or set `artifact`? It also looks like GCE has the same error. Or else I am missing how this works :)

Reply:
It is called `archive`, because the output is an archive (in GCE case at least). Same goes for XZ compressed images. The pipeline name does not matter as long as the image type specifies the same as the name of the pipeline to export. The name has no meaning other than the fact that we need to pass to `osbuild` the pipeline name which result should be exported (technically it could be even more pipeline names than just one, but composer is not able to handle it nor our image definitions).

Reply:
What I mean is this function returns `archive`, and in these 2 blocks nothing is setting that to anything so it's going to remain set to `nil`. If compression is selected then it operates on `artifactPipeline` which also isn't set in these blocks. So how does this function return anything for these platform cases? It looks to me like it will just return `nil`.

Reply:
Brian's right: the `artifactPipeline` is defined outside the switch block and is meant to be set based on each case.  It doesn't cause any issues because we never use the return value of the function.  It's meant to define the export pipeline to pass to osbuild but we never wired it up properly and still use the old method where we define the export pipeline name on the image type.

This should be fixed though because we'll probably be wiring it up soon.